Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx

```diff
@@ -383,8 +383,9 @@ export default class AccessContainer extends React.Component<IProps, IState> {
         }
     };
 
-    private _updateAccessFromDetails = (access: IAccess) => {
-        const index = this.state.accesses.findIndex(x => x.id === access.id);
+    private _updateAccessFromDetails = (access: IAccess, id?: number) => {
```

Comment:
Why this strategy of adding a new id param instead of just doing something like `const id = access?.id`?  Seems like it would tell you the same info

Reply:
@srkirkland it's because if what they're trying to look at has been deleted, `access` will be null. but i still want the id so i can update the state, so i send that separately

Reply:
ooh so it's not the access.id really, it's the old deleted accessId that shows up there.  ok 🚢